build.yaml

```diff
@@ -1204,3 +1203,55 @@ job_type:
                    *package_jenkins_osx_yosemite_box ]
           }
       timeout: 120
+
+#-----------------------------------------------------------------------------#
+# View definitions below this point
+#-----------------------------------------------------------------------------#
+
+views:
+  trial:
+    description: 'All trial jobs'
+    regex: 'run_(?i)trial_.*.*'
```

Comment:
Sounds good. I guess the thing is that as a flocker developer, I don't know what I'd use it for. (I have no objections against having it).